### PR TITLE
Add detailed Windows SSH instructions

### PR DESCRIPTION
**Pull request by lbel**
These have been tested on Windows XP, but contain nothing that shouldn't work on newer versions of Windows.

### CODE REVIEW

File: 00-prerequisites.md [hunk 1]
```diff
@@ -88,3 +88,24 @@ Proxies uploaded:
       If either of the above two commands do not work please email `lhcb-starterkit@cern.ch`
 
 If you can successfully execute all of the above steps you are ready to go for the workshop.
+
+> ## Windows-specific Instructions {.callout}
+>
+> On Windows, some steps are required before you can connect via SSH.
+>
+> Setup steps (you only have to perform this once):
+> 1. Download the [Xming installer](http://sourceforge.net/projects/xming/files/latest/download).
```
Comment (betatim) [inline 00-prerequisites.md:97]: add a space here so the listing works properly

**Comment by betatim** [comment]
Thanks a lot for this!!

The instructions look good to me, but then I haven't used windows in about a decade :-/

I would suggest we merge it unless someone is against it.

/cc @kdungs @alexpearce

**Comment by kdungs** [comment]
I can try this at home on my Windows but from here it looks good.

**Comment by betatim** [comment]
Next time can we wait for the formatting to be fixed? How do we repair this now?

**Pull request merged**